Build/Customizations/Projects.props

```diff
@@ -195,6 +195,7 @@
     <ReleaseProjectFiles Include="$(SolutionDirectory)\Remotion\ObjectBinding\Security\ObjectBinding.Security.csproj"/>
     <ReleaseProjectFiles Include="$(SolutionDirectory)\Remotion\ObjectBinding\Validation\ObjectBinding.Validation.csproj"/>
     <ReleaseProjectFiles Include="$(SolutionDirectory)\Remotion\ObjectBinding\Web\ObjectBinding.Web.csproj"/>
+    <ReleaseProjectFiles Include="$(SolutionDirectory)\Remotion\ObjectBinding\Web.ClientScript\ObjectBinding.Web.ClientScript.csproj"/>
```

Comment:
the updates to the nuspec file are missing. It is possible, that we will need dedicated files for web and objectbinding.web instead of being able to to use the shared file we have now.